assets/js/Viewer.test.js

```diff
@@ -3,15 +3,14 @@ import { mount } from 'enzyme';
 
 import Viewer from './Viewer';
 
-test('does not show messages that have expired', () => {
-  const now = Date.now();
-  const expired = new Date(now).toLocaleString();
-  const fresh = new Date(now + 5000).toLocaleString();
-  const signs = { signID: [{ text: 'Alewife 1 min', duration: fresh }, { text: 'Alewife 3 min', duration: expired }] };
+test('Shows all signs for a line', () => {
```

Comment:
I see. You have the line showing/hiding test here. I think it makes more sense in `Line` since that's where the logic actually is.

However, I think something like this might make sense as a `ViewerApp` test, as something of an integration test. If you look at the [docs](https://airbnb.io/enzyme/docs/api/mount.html#full-rendering-api-mount) for our testing library, there's an example of `find(...).simulate('click')`. It would be nice to have a test of that, to exercise your `onClick` functions.

Reply:
yeah, ill add something like that and move the other tests around

Reply:
I feel like this test can be removed now since it's covered by the tests in Sign and Line. Is there any other functionality being tested here?

Reply:
oh, i actually had removed it but i guess it didnt save - ill push that up